Lib/test/test_ntpath.py

```diff
@@ -806,6 +808,9 @@ def test_abspath(self):
         tester('ntpath.abspath("C:\\spam. . .")', "C:\\spam")
         tester('ntpath.abspath("C:/nul")',  "\\\\.\\nul")
         tester('ntpath.abspath("C:\\nul")', "\\\\.\\nul")
+        tester('ntpath.abspath("Z:spam")', r"Z:\spam")
+        self.assertEqual(ntpath.abspath("Z:\x00"), "Z:\\\x00")
```

Comment:
These two tests can't assume that drive "Z:" doesn't exist or that it doesn't have a non-root working directory if it does exist. For example, compare the result to `join(abspath("Z:"), "spam")`.

Reply:
Hmm, we actually just want it to return an absolute path:

```suggestion
        self.assertTrue(ntpath.isabs(ntpath.abspath("C:spam")))
        self.assertTrue(ntpath.isabs(ntpath.abspath("C:\x00")))
```